node/pkg/txverifier/utils.go

```diff
@@ -23,6 +23,10 @@ const (
 func extractFromJsonPath[T any](data json.RawMessage, path string) (T, error) {
 	var defaultT T
 
+	if data == nil {
+		return defaultT, fmt.Errorf("supplied JSON data is nil")
```

Comment:
Later in this function, could you modify the error messages `return defaultT, fmt.Errorf("can't convert to type T")`? It will always just return the letter `T`, not the actual type